### PR TITLE
Maintenance page

### PR DESCRIPTION
**Pull request by Andresws12**

### CODE REVIEW

**Comment by Andresws12** [comment]
Untitled
@gonzalo-plaza Maybe with the picture you can understand jajajaja I lik but i there are a lot space with in title and subtitle,  i would say that if the text it's more align on the center looks better and move  the copy to bottom

**Comment by gonzalo-plaza** [comment]
Perfect. I think that I have understand that do you want explain, I go to try and after you say me. Thank You Brotheer

**Comment by Andresws12** [comment]
image
@gonzalo-plaza  could do you check the "acciÃ³n"  text please?
and... on my  laptop is'nt showing okay, maybe the image is too large
image

**Comment by gonzalo-plaza** [comment]
@Andresws12 perfect, I am correcting the fails. Thank you!!

**Comment by Andresws12** [comment]
@gonzalo-plaza for me it's okay, maybe @PVillasana @JuanJosePG and @gmbyana cand check the page and code and give you feedback

File: maintenance.html [hunk 1]
```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE hmtl>
+<html lang="es">
+    <head>
+        <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width, user-scalable=no" />
+        <title>Mantenimiento</title>
+        <style type="text/css">
+            * {
+                font-family: Arial, Helvetica, sans-serif;
```
Comment (Andresws12) [inline maintenance.html]: @gonzalo-plaza  This is all fonts, maybe you can check on css tricks https://css-tricks.com/snippets/css/system-font-stack/
font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Segoe UI Emoji", "Segoe UI Symbol", Roboto, Helvetica, "Helvetica Neue", Oxygen-Sans, Ubuntu, Cantarell, Arial, sans-serif, "Apple Color Emoji";

**Comment by gonzalo-plaza** [comment]
I have applied the fonts of the operating systems, but in the link that you have recommend me, it say that que can change "-apple-system" and "BlinkMacSystemFont" for the new universal font "system-ui". But I haven´t apply this because it say that is universal for safary and chrome, and I don´t know if is a good option. I also added the "Tahoma" font, I know Windows XP is obsolet but I think that still can exist any some PC with XP. Correct me if is no ok. Thank you!!

**Pull request merged**